tests/app/test_metrics.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+from tests.helpers import BaseApplicationTest
+
+
+def load_prometheus_metrics(response_bytes):
+    return dict(re.findall(rb"(\w+{.+?}) (\d+)", response_bytes))
+
+
+class TestMetrics(BaseApplicationTest):
+
+    def test_metrics_page_accessible(self):
+        metrics_response = self.client.get('/_metrics')
+
+        assert metrics_response.status_code == 200
+
+    def test_metrics_page_contents(self):
+        metrics_response = self.client.get('/_metrics')
+        results = load_prometheus_metrics(metrics_response.data)
+        assert (
+            b'http_server_requests_total{code="200",host="localhost",method="GET",path="/"}'
```

Comment:
On the API pull request this test asserted the port as well (and `/_metrics` as the path):

```
assert `b'http_server_requests_total{code="200",host="127.0.0.1:5000",method="GET",path="/_metrics"}'` in results
```

Should we be doing similar here?

Reply:
For whatever reason the search-api metrics don't contain the port, have changed that test to look for `/_metrics` on the metrics page as that will definitely exist